src/deploy/NVA_build/NooBaa.Dockerfile

```diff
@@ -150,6 +152,8 @@ EXPOSE 26050
 # Run as non root user that belongs to root 
 USER 10001:0
 
+ENV LD_PRELOAD /usr/lib64/libjemalloc.so.2
```

Comment:
Not here, Place it next to all the ENV (Line 53)
In this point you are not root anymore.

Reply:
it has to be after the install otherwise the file is not there and the env failes. should I move before 153?

Reply:
yes

Reply:
done